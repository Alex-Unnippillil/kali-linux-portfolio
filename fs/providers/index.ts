export * from './types';
export { MemoryFileSystemProvider } from './memory';
export { ZipFileSystemProvider } from './zip';
