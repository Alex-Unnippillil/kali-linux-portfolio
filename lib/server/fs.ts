import 'server-only';
import fs from 'fs';

export const promises = fs.promises;

export default fs;
