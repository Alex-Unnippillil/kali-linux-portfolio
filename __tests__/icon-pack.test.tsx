import fs from 'fs';
import path from 'path';

describe('adwaita icon pack', () => {
  it('matches expected file list', () => {
    const dir = path.join(process.cwd(), 'public', 'icons', 'adwaita');
    const files = fs.readdirSync(dir).sort();
    expect(files).toMatchSnapshot();
  });
});
