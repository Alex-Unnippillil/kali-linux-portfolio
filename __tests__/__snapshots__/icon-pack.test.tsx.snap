// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`adwaita icon pack matches expected file list 1`] = `
[
  "audio-volume-medium-symbolic.svg",
  "battery-good-symbolic.svg",
  "network-wireless-signal-good-symbolic.svg",
  "network-wireless-signal-none-symbolic.svg",
]
`;
